packages/flutter/lib/src/animation/curves.dart

```diff
@@ -45,8 +49,12 @@ class Linear extends Curve {
 
 /// A sawtooth curve that repeats a given number of times over the unit interval.
```

Comment:
this should say what direction the saw tooth is in. Presumably, it goes from 0.0 to 1.0 `count` times, with a discontinuous drop from 1.0 back to 0.0 between each iteration.